packages/airnode-node/src/evm/handlers/process-transactions.ts

```diff
@@ -33,27 +33,17 @@ export async function processTransactions(
   // =================================================================
   // STEP 3: Get the latest gas price
   // =================================================================
-  const gasPriceOptions = { provider: state2.provider, chainOptions };
-  const [gasPriceLogs, gasTarget] = await getGasPrice(gasPriceOptions);
-  logger.logPending(gasPriceLogs, baseLogOptions);
-
-  if (!gasTarget) {
-    logger.error('Unable to submit transactions without gas price. Returning...', baseLogOptions);
-    return state2;
-  }
-
-  if (chainOptions.txType === 'eip1559') {
-    const gweiMaxFee = utils.weiToGwei(gasTarget.maxFeePerGas!);
-    const gweiPriorityFee = utils.weiToGwei(gasTarget.maxPriorityFeePerGas!);
-    logger.info(
-      `Gas price (EIP-1559) set to a Max Fee of ${gweiMaxFee} Gwei and a Priority Fee of ${gweiPriorityFee} Gwei`,
-      baseLogOptions
-    );
-  } else {
-    const gweiPrice = utils.weiToGwei(gasTarget.gasPrice!);
-    logger.info(`Gas price (legacy) set to ${gweiPrice} Gwei`, baseLogOptions);
-  }
+  const [logs, gasPrice] = await getGasPrice(state2.provider, chainOptions.gasPriceOracle);
+  logger.logPending(logs, baseLogOptions);
 
+  const gasPriceWithType =
+    chainOptions.txType === 'eip1559'
+      ? { type: 2, maxFeePerGas: gasPrice, maxPriorityFeePerGas: gasPrice }
```

Comment:
Is it ok to use the same number for `maxFeePerGas` and `maxPriorityFeePerGas`?

Reply:
It's common practice to set `maxFeePerGas` and `maxPriorityFeePerGas` to the gasPrice _if_ it's a txType 2 chain and the user has specified the older-style gas bid - but I don't know if that's what's going on here?

Reply:
> It's common practice to set maxFeePerGas and maxPriorityFeePerGas to the gasPrice if it's a txType 2 chain and the user has specified the older-style gas bid - but I don't know if that's what's going on here?

This was the idea as `ethers` throws an error if we specify `type: 2` and try to set the `gasPrice` field.